### PR TITLE
Update Readme for Migration to provider v1.0.0

### PR DESCRIPTION
**Pull request by outscale-toa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,64 @@
 - [Go](https://golang.org/doc/install) 1.13 (to build the provider plugin)
 
 
+## Migration to v1
+
+> [!WARNING]
+>
+> Before you begin using the ```v1``` binary on your Terraform code, make sure to back up your state file. If you are using a local state file, you can simply make a copy of your terraform.tfstate file in your project directory.
```
Comment (outscale-mgo) [inline README.md:20]: > Before you begin using the ```v1``` binary on your Terraform code, make sure to back up your state file. If you are using a local state file, you can simply make a copy of your terraform.tfstate file in your project directory.
 
I would split that line, so we can see the part about backuping stuff better.
 
  > Before you begin using the ```v1``` binary on your Terraform code, make sure to back up your state file. 
  > If you are using a local state file, you can simply make a copy of your terraform.tfstate file in your project directory.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,64 @@
...
+> If you are using a remote backend such as an S3 bucket, make sure that you follow the backup procedures for the backend and that you exercise the restore procedure at least once.
```
Comment (outscale-mgo) [inline README.md:21]: > If you are using a remote backend such as an S3 bucket, make sure that you follow the backup procedures and that you exercise the restore procedure at least once.

File: README.md [hunk 3]
```diff
@@ -13,6 +13,64 @@
...
+
+provider "outscale" {
+  # Configuration options
+}
+```
+
+```sh
+terraform init -upgrade
+```
+
```
Comment (outscale-mgo) [inline README.md:45]: can you explain with a few small line, what the sed are doing, pelase.

**Review by outscale-mgo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,22 @@
...
+
+* Error in attributes value for healthy_threshold ([GH-450](https://github.com/outscale/terraform-provider-outscale/issues/450))
+
+FEATURES:
+---------
+
+* Enable adding Tag on boot disk ([GH-110](https://github.com/outscale/terraform-provider-outscale/issues/110))
+* Enable linked multiple flexible GPUs to VM ([GH-220](https://github.com/outscale/terraform-provider-outscale/issues/220))
+
+IMPROVEMENT:
```
Comment (outscale-hmi) [inline CHANGELOG.md:14]: IMPROVEMENTS*:

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,22 @@
...
+-----------
+* Rename ``` outscale_volumes_link ``` to ``` outscale_volume_link ```
+* Upgrade to Terraform-plugin-sdkv2 and terraform-plugin-framework ([GH-150](https://github.com/outscale/terraform-provider-outscale/issues/150))
+* Somes refactoring
```
Comment (outscale-hmi) [inline CHANGELOG.md:18]: Some*

File: README.md [hunk 6]
```diff
@@ -13,13 +13,86 @@
 - [Go](https://golang.org/doc/install) 1.22.0 (to build the provider plugin)
 
 
+## Migration to v1
+
+> [!WARNING]
+>
+> Before you begin using the ```v1``` binary on your Terraform code, make sure to back up your state file!
+> If you are using a local state file, you can just copy of your terraform.tfstate file in your project directory.
```
Comment (outscale-hmi) [inline README.md:21]: you can just copy or make copy of **

File: README.md [hunk 7]
```diff
@@ -13,13 +13,86 @@
...
+}
+```
+
+```sh
+terraform init -upgrade
+```
+
+### Step 2: Edit terraform state and configuration files
+
+Somes block types changed in terraform state, the following script will delete those blocks.
```
Comment (outscale-hmi) [inline README.md:50]: Some* block types changed in the Terraform state. The following script will delete those blocks.

**Review by outscale-hmi — APPROVED** [review]

**Review by outscale-rce — APPROVED** [review]

**Review by outscale-rce — APPROVED** [review]

**Review by outscale-mdr — APPROVED** [review]

**Review by outscale-mgo — APPROVED** [review]

**Pull request merged**